share/spice/github/github.js

```diff
@@ -6,73 +6,96 @@
-                    sourceUrl: 'https://www.github.com/search?q=' +  encodeURIComponent(query),
-                    sourceName: 'GitHub'
+                    sourceUrl: 'https://www.github.com/' + results.full_name,
+                    sourceName: 'GitHub',
+                    rerender: [ 'description', 'image' ]
                 },
                 templates: {
-                        group: 'text',
-                        detail: false,
-                        item_detail: false,
+                        group: 'info',
                         options: {
-                            footer: Spice.github.footer
-                        },
-                        variants: {
-                            tile: 'basic4'
+                            moreAt: true
                         }
                     },
-                normalize: function(item) {
-                    if (item.size === 0) {
-                        return;
-                    }
-                    return {
-                        title: item.name,
-                        subtitle: item.owner.login + "/" + item.name,
-                        url: item.html_url,
-                        pushed_at: moment(item.pushed_at).fromNow()
-                    };
-                },
                 relevancy: {
                     primary: [
-                        { key: 'description', match: /.+/, strict: false } // Reject things without a description.
+                        { required: 'description' }
                     ]
                 },
-                sort_fields: {
-                    watchers: function(a, b) {
-                        var x = a.watchers;
-                        var y = b.watchers;
-                        return ((x < y) ? 1 : ((x > y) ? -1 : 0));
+                onItemShown: function(item){
+                    if(!fatheads[key]){
+                        return;
                     }
-                },
-                sort_default: 'watchers'
+                    var apiReq = fatheads[key].url + "&o=json";
+                    
+                    // check if abstract is set already to avoid calling
+                    // api again if we switch tabs
+                    if(!fatheads[key].abstract){
+                        $.getJSON(apiReq,
+                            function(data){
+                                fatheads[key].image = data.Image;
+                                fatheads[key].abstract = data.Abstract;
+
+                                if(fatheads[key].abstract){
+                                    item.set({ 
+                                        description : fatheads[key].abstract, 
+                                        image: fatheads[key].image
+                                    });
+                                }
+                            });
+                    }
+                }
             });
+            
+            function get_days(date) {
```

Comment:
Moment can do this for you 😃  `moment(date).fromNow()`

http://momentjs.com/docs/#/displaying/fromnow/